lfads/lfads.py

```diff
@@ -769,8 +768,6 @@ def encode_data(dataset_bxtxd, enc_cell, name, forward_or_reverse,
     # Variational Lower Bound on posterior, p(z|x), plus reconstruction cost.
     # KL and reconstruction costs are normalized only by batch size, not by
     # dimension, or by time steps.
-    kl_cost_g0_b = tf.zeros_like(batch_size, dtype=tf.float32)
-    kl_cost_co_b = tf.zeros_like(batch_size, dtype=tf.float32)
```

Comment:
I think the appropriate fix is to replace lines 779-780 (the `kl_cost_g0_b = 0.0` and `kl_cost_co_b = 0.0`) with these two lines that are proposed to be deleted. We want the variables `kl_cost_g0_b` and `kl_cost_co_b` to be batch-sized initialized (not be scalars)